wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/README.md

```diff
@@ -0,0 +1,71 @@
+# WordPress.org Openverse Theme
+
+This theme loads Openverse in an `iframe` under the WordPress.org site.
+
+## Development
+
+Follow these steps to set up a local playground for the theme:
+
+0.  Install all the prerequisites.
+
+    1.  **Required:** Node.js.
+    2.  **Recommended:** Docker (to use the automatic setup)
+
+1.  Build the parent theme WordPress.org theme.
+
+    1.  `cd` into the parent theme directory at `../wporg`.
```

Comment:
I think it would be better to have the full path to the directory from content root.

Reply:
It's actually quite deep down in the folder tree so the full path is considerably long. Also since the `README.md` is itself not in the root of the repo, the person reading it will quite likely already be in the `wporg-openverse` folder. From there, a relative path will be more helpful imo.

Reply:
Why not both? Resolved in 2dc4ac2.